elements/pfe-band/demo/index.html

```diff
@@ -291,7 +336,7 @@ <h2>Short content band</h2>
       </header>
       <p>Short content sadipscing elitra.</p>
       <pfe-cta pfe-priority="primary" id="test-cta">
-        <a href="#">Learn more</a>
+        <a href="#">This CTA has overrides</a>
```

Comment:
I'm not seeing the overrides on the demo page: https://deploy-preview-895--happy-galileo-ea79c4.netlify.app/elements/pfe-band/demo/#test-custom

Reply:
hmm, i see the padding adjusted
![image](https://user-images.githubusercontent.com/456863/88802429-101ff800-d179-11ea-8d9f-8fa44af7e1b9.png)